.ci/scripts/tropo/compare_tropo_products.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# Script used to orchestrate the pairwise comparison of output and expected
+# TROPO products. Each individual pair of products is compared using the
+# tropo_comparison.py script
```

Comment:
I don't think this `tropo_comparison.py` script exists? Should change to something like "the opera_tropo validate workflow"